build.sbt

```diff
@@ -2,16 +2,22 @@ import scala.sys.process._
 
 def crossPlugin(x: sbt.librarymanagement.ModuleID) = compilerPlugin(x.cross(CrossVersion.full))
 
-val compilerPlugins = List(
-  crossPlugin("org.polyvariant" % "better-tostring" % "0.3.15"),
-  crossPlugin("org.typelevel" % "kind-projector" % "0.13.2"),
-)
+val compilerPlugins =
+  libraryDependencies ++= List(
+    crossPlugin("org.polyvariant" % "better-tostring" % "0.3.15")
+  ) ++ (if (scalaVersion.value.startsWith("3"))
+          Nil
+        else
+          List(
+            crossPlugin("org.typelevel" % "kind-projector" % "0.13.2")
+          ))
```

Comment:
Thought I was gonna do the LSP in Scala 3, but I was getting too annoyed by the tooling.